### ISSUE TITLE
share button url query issue  

### ISSUE THREAD

**Issue by sanjay7178** [Sarath191181208/vitap-timetable-scheduler#8]
![image](https://github.com/user-attachments/assets/9a9a1d62-7a31-4049-a584-b0bb58cc8f69)

**Comment by sai-krishna-git** [comment]
Hi, Can you explain about the issue and steps to generate?

**Comment by Sarath191181208** [comment]
@sanjay7178 can you share the url that gave you this not found error.

**Comment by sanjay7178** [comment]
[https://vitap-time-table-scheduler.vercel.app/share?v=1&sem=FALL-2024-25&data=LIKAYAyMAoCACgMDQA4YA8AkAYCA](https://vitap-time-table-scheduler.vercel.app/share?v=1&sem=FALL-2024-25&data=LIKAYAyMAoCACgMDQA4YA8AkAYCA)

@sai-krishna-git @Sarath191181208 check it

**Comment by Sarath191181208** [comment]
Well, the issue is that this current semester data isn't present in the application therefore the application is giving an error. This is an expected error but I think it would be better to have a error page for better understanding.

**Comment by Sarath191181208** [comment]
This issue has been fixed by merging #9 . The issue was `vercel` not redirecting the /share requests to the react router therefore not showing the page. This is fixed by adding this to **vercel.json**

```json
{
  "rewrites":[
    {"source": "/", "destination": "/"},
    {"source": "/share", "destination": "/"}
  ],
}
```
